### DISCUSSION TITLE
Error : Laravel 11

### DISCUSSION THREAD

**Discussion by stuckfakur** [supabase/supabase#23623] [Show and tell]
``` SQLSTATE[3F000]: Invalid schema name: 7 ERROR:  no schema has been selected to create in at character 14 (Connection: pgsql, SQL: create table "migrations" ("id" serial not null primary key, "migration" varchar(255) not null, "batch" integer not null))```

I was input the configuration at env 
DB_CONNECTION=pgsql
DATABASE_URL=postgres://postgres.refrence:password@aws-0-ap-southeast-1.pooler.supabase.com:5432/postgres
DB_PASSWORD=password

and on database.php was change like as a doc

but while i migrate showing error like at above

**Comment by arifd11** [comment]
In app/config/database.php change 
'search_path' => 'laravel' -> 'search_path' => 'public'

**Reply by EfrenPech** [reply]
Thank you very much it worked

**Comment by improved-software** [comment]
The `public` schema is less secure.  ❌

Add a `laravel` schema in the SQL Editor  ✔
```sql
CREATE SCHEMA laravel
```
